sdcm/nemesis/monkey/network.py

```
@@ -0,0 +1,570 @@
+        target_node.ip_address,
+        target_node.scylla_network_configuration.device,
+    )
+    now = time.time()
+    results = prometheus_stats.query(query=query, start=now - 600, end=now)
+    assert results, "no results for node_network_receive_bytes_total metric in Prometheus "
+    received_bytes_over_time = [float(avg_rate) for _, avg_rate in results[0]["values"]]
+    avg_bitrate_per_node = (received_bytes_over_time[-1] - received_bytes_over_time[0]) / 600
+    avg_mpbs_per_node = avg_bitrate_per_node / 1024 / 1024
+
+    if avg_mpbs_per_node > 10:
+        min_limit = int(round(avg_mpbs_per_node * 0.30))
```

Ruff (0.15.14)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)